documentation/modules/managing/proc-cluster-recovery-volume.adoc

```diff
@@ -111,7 +111,7 @@ spec:
     *apiVersion: v1*
     *kind: PersistentVolumeClaim*
     *name: data-0-my-cluster-kafka-2*
-    *namespace: myproject*
+    *namespace: my-project*
```

Comment:
Should we stick with `myproject` everywhere? That is what we use int he installation files etc.

Reply:
There was a mix of the two, but I can move all to myproject.

Reply:
Thanks